spring-cloud-kubernetes-client-discovery/src/main/java/org/springframework/cloud/kubernetes/client/discovery/reactive/KubernetesInformerReactiveDiscoveryClientAutoConfiguration.java

```diff
@@ -135,4 +143,15 @@ KubernetesInformerDiscoveryClient kubernetesClientInformerDiscoveryClient(
 				serviceInformer, endpointsInformer, properties);
 	}
 
+	@Bean
```

Comment:
same as for blocking implementation, create the underlying client depending on the selective namespaces being enabled or not.